### ISSUE TITLE
Using weekday seems to give incorrect results.

### ISSUE THREAD

**Issue by scottbeaumont** [gastromatic/calendar-date#210]
According to the documentation found [here](https://github.com/gastromatic/calendar-date#dayoftheweek), I expect the following test should pass since January 1st, 2023 is a Sunday.

```tsx
test('simple calendar test', () => {
     expect(new CalendarDate(`2023-01-01`).weekday).toBe(7)
})
```

Unfortunately, it fails with 
```
simple calendar test
expect(received).toBe(expected) // Object.is equality
    Expected: 7
    Received: 6
```

Am I missing something, or is this unexpected?

**Comment by co-sic** [comment]
Hey @scottbeaumont thank you for reporting this issue! I just tested it, and this is a bug in the constructor. Your current timezone is affecting the weekday, that is why it is working for me (Europe timezone) and not for you (I guess you are in the USA?). I will fix this as soon as possible and get back to you!

**Comment by co-sic** [comment]
@scottbeaumont solved in #212

**Comment by scottbeaumont** [comment]
@co-sic Thanks for resolving and thanks for the package.  It's appreciated.
